docs/user-guide-policy-configure-linux-bond-with-vlans.md

```diff
@@ -27,11 +27,6 @@ spec:
     - name: bond0
       type: bond
       state: up
-      ipv4:
```

Comment:
What's ipv4 by default?  enable: false ? maybe is better to be explicit here.

Reply:
It is disabled unless said otherwise https://github.com/nmstate/nmstate/blob/master/libnmstate/nm/ipv4.py#L52 (if I read it right).

I'd prefer to keep the docs small and attractive. I can make this use DHCP if you think it makes more sense. I don't have a strong opinion about this.

Reply:
We can have a chapter related to ip configuration and use at the other parts DHCP, so I am ok with using dhcp here also most of the people is going to use DHCP.